my-skill/SKILL.md

```
@@ -0,0 +1,454 @@
+3. check_a11y({ html: "..." })
+   → { score: 100, passes: [...], issues: [] }
+
+4. generate_theme({ prompt: "dark teal round" })   ← optional
+   → :root { ... }
+```
+
+---
+
+## 13. Emmet Quick Reference
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)